Backend/UnitTesting/TestingUtils.cs

```diff
@@ -90,7 +90,46 @@ public Session CreateSessionInDb(Session session)
                 return session;
             }
         }
-        
+
+        public PasswordReset CreatePasswordResetInDB()
+        {
+            PasswordReset pr = new PasswordReset
+            {
+                PasswordResetID = new Guid(),
+                ResetToken = "",
+                UserID = new Guid(),
+                ExpirationTime = DateTime.Now.AddMinutes(5),
+                ResetCount = 0,
+                Disabled = false
+            };
+            return CreatePasswordResetInDB(pr);
+        }
+
+        public PasswordReset CreatePasswordResetInDB(PasswordReset resetToken)
+        {
+            using (var _db = new DatabaseContext())
+            {
+                _db.Entry(resetToken).State = System.Data.Entity.EntityState.Added;
+                _db.SaveChanges();
+                return resetToken;
+            }
+        }
+
+        public PasswordReset CreatePasswordResetObject()
+        {
+            PasswordReset pr = new PasswordReset
+            {
+                PasswordResetID = new Guid(),
+                ResetToken = "",
+                UserID = new Guid(),
+                ExpirationTime = DateTime.Now.AddMinutes(5),
+                ResetCount = 0,
+                Disabled = false
+            };
+            return pr;
+        }
+
+        /*
         public Service CreateServiceInDb(bool enabled)
```

Comment:
Some of these unused utils and tests have been removed in the bug branch. Expect some merge conflicts